tests/caasp/first_boot.pm

```diff
@@ -59,6 +59,8 @@ sub run {
         }
         # End of workaround
     }
+
+    record_info "RZP ping should fail" if script_run "ping -c1 updates.suse.com";
```

Comment:
what exactly are you testing with that? First boot it *after* the installation.

Reply:
same as before, just to check that I can't ping 11.22.33.44